paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass_tester.cc

```diff
@@ -374,6 +395,22 @@ TEST(CpuQuantizeSquashPass, conv_dequant_more_than_one_op_after_conv) {
                 remove_nodes);
 }
 
+// from
+// a->fc->b
+// b->Dequant1->c
+// c->Concat1->d
+// to
+// a->fc->c->Concat->d
+TEST(CpuQuantizeSquashPass, fc_dequant_only_one_output) {
+  auto scale_out = 1.0f;
+  auto scale = 1.2345f;
+  auto use_mkldnn = true;
+  // remove 2 nodes: b, Dequant1
+  auto remove_nodes = 2;
+  CountNodeTest(BuildFcDequantConcatProgramDesc(use_mkldnn, scale_out, scale),
+                remove_nodes);
+}
```

Comment:
Please, add verification that the `force_fp32_output` attribute is set to `true` in `fc`.